client/a8c-for-agencies/README.md

```diff
@@ -0,0 +1,11 @@
+# A8C for Agencies
+
+## Getting Started
+
+You can try out the user-side of Calypso on https://agencies.automattic.com/ or install and run it locally:
```

Comment:
```suggestion
You can try out the front-end of A4A on https://agencies.automattic.com/ or install and run it locally:
```
"user-side" sounds like a clunky way of saying "front-end" and "Calypso" should probably be replaced by "A4A"